src/memory/GameService.java

```diff
@@ -0,0 +1,7 @@
+package memory;
+
+public interface GameService {
```

Comment:
Please, try to create new packages in order to decouple the logic, e.g. a new package to UI components

Reply:
done